clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/WgsUbamElasticsearchDAOSpec.scala

```diff
@@ -23,6 +24,7 @@ class WgsUbamElasticsearchDAOSpec
   it should "updateWgsUbamMetadata" in {
     val key = ModelWgsUbamKey("barcodeURGM1", 2, "library3", Location.GCP)
     val metadata = ModelWgsUbamMetadata(
+      clioId = Some(ClioUUIDGenerator.getUUID()),
```

Comment:
Is `clioId` required here? I thought that the update API didn't need an ID as input.

Reply:
This also makes me wonder, should we let clients modify `clioId` using update? That seems like that would break our model.

Reply:
The UUID is assigned in the WgsUbamService. Since this test is testing the layer beneath that, the DAO, there should already be a UUID assigned. Clients are not prohibited from supplying a clioId for an upsert, but it will be overridden before being added to ElasticSearch.

Reply:
OK, I missed that this was testing the DAO layer, not the API layer.